assets/stylesheets/bootstrap/_badges.scss

```diff
@@ -18,7 +18,7 @@
   background-color: $badge-bg;
   border-radius: $badge-border-radius;
 
-  // Empty badges collapse automatically
+  // Empty badges collapse automatically (not available in IE8)
```

Comment:
Please restore the comment to:

`  // Empty badges collapse automatically`

I do not want to re-introduce any speak about IE8